Incorrect filepath for app.js in created docker image

#### Description:
When creating the Docker image, there is an issue where `app.js` is placed in the wrong directory. We manually converted and generated the `app.js` file in `/lib/app.js`. However, this caused issues with module imports, specifically failing to resolve paths such as `./container` from within `app.js`.

#### Steps to Reproduce:
1. Build the Docker image as per the current instructions.
2. The `app.js` file is generated in an incorrect path (`/lib/app.js`).
3. This leads to import errors like `Cannot find module './container'` when running the application.

#### Expected Behavior:
The `app.js` file should be correctly placed in the appropriate directory, and all internal module imports should resolve without errors.

#### Actual Behavior:
- The `app.js` file is generated in `/lib/app.js`.
- Import paths within `app.js` fail (e.g., `./container` and other dependencies).

#### Additional Information:
- I have attached screenshots below showing the errors and our attempts to fix the issue.
- We tried modifying the path manually, but the import references still failed.

#### Attempts to Fix:
- Manually placed the `app.js` file in `/lib/app.js`.
- Tried adjusting the Dockerfile and build configurations, but the imports still failed.

#### Screenshots:
`app.js` not found:
![image](https://github.com/user-attachments/assets/0a5864e2-6027-403c-80f1-9d7c078ed0aa)

`app.js` gets created in work directory (/home/app):
![image](https://github.com/user-attachments/assets/36e52693-627b-4fbe-b2c6-dd0017617246)

Attempted fix failed by manually copying the `app.js` to the required destination:
![image](https://github.com/user-attachments/assets/aee30aa8-eb71-4525-b224-b3704ec54ec3)
![image](https://github.com/user-attachments/assets/557edd53-bc9f-4397-917c-f77b64e9a6f1)

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,9 @@
-FROM  node
+FROM  node:18
 
 COPY . /home/app
-RUN cd /home/app && npm i
+RUN cd /home/app \
+    && npm i \
+    && npm run build
 
 EXPOSE 8100
 WORKDIR /home/app

